src/components/storefront/ProductDetails/QuantityPicker.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
```

Comment:
it may make sense to move this file to the path `src/components/storefront/QuantityPicker/QuantityPicker.js`

cc @hongyi-chen you can just do this on your branch I think